src/components/tabs-board/styles.ts

```diff
@@ -0,0 +1,37 @@
+import { styled } from "@mui/material/styles";
+import {Box, BoxProps, Typography} from "@mui/material";
+
+interface StyledNumberCircleProps extends BoxProps {
+	isBeforeActive?: boolean;
+}
+
```

Comment:
We do not use interfaces. Change to type
```suggestion
type StyledNumberCircleProps = {isBeforeActive?: boolean} & BoxProps
```